releasing/lagom.md

```diff
@@ -120,21 +120,45 @@ Merge your release pull request and deploy the updated website from `vegemite`.
 *NOTE*: the command logs its output to `~/logs/lagom-nightly-deploy-master-1532020306.log` or similar. You need
 to find the most recent file to `tail` if you can to see what is happening.
 
-## Update the g8 templates and example projects
+## Update the downstream projects and other deps
```

Comment:
You break the TOC at the top as well as the release checklists, both the past ones and the future ones based on the template.

If we want to avoid this we should use a more stable identifier for these sections.

Reply:
```suggestion
## Update the g8 templates and example projects
```

Reply:
thanks @dwijnand ! Didn't think of that.

I've reverted and we can fix on a separate PR